### ISSUE TITLE
Logo proposal

### ISSUE THREAD

**Issue by dee-y** [bailabs/tailpos#32]
Hello, i am an open source graphic designer who loves contributing to the open source community. I like your projects idea and goal. I noticed you are without a logo for your project, would you like me to contribute one to your project?

**Comment by ccfiel** [comment]
ok sure! :)

**Comment by dee-y** [comment]
Great! Will have some designs for you to have a look at soon.

**Comment by ccfiel** [comment]
thanks!

**Comment by dee-y** [comment]
I would love some insight to understand the project name [TailPOS] , i already know what it does. What does the Tail mean? I understand the POS for point of sale.

**Comment by ccfiel** [comment]
tail comes the word retail :)

**Comment by dee-y** [comment]
Oh i see, very crafty :)

**Comment by dee-y** [comment]
Hi guys, came up with the lettering idea of "T" and "P", and also tried to incorporate a POS receipt into it. What do you think? 

![tailpos-02](https://user-images.githubusercontent.com/36637989/57987226-66be7c00-7a76-11e9-87a9-e80e1d26a573.jpg)
![tailpos-03](https://user-images.githubusercontent.com/36637989/57987227-66be7c00-7a76-11e9-8fff-95cad41763b3.jpg)

**Comment by ccfiel** [comment]
@dee-y Thanks! this is great! I love the logo do you a svg file for this? We will try to incorporate it to the app. :)

**Comment by dee-y** [comment]
Okay great!. Do you have any color suggestions? Or you like it black like this? And I think it's better without the receipt. Let me know your thoughts so i make final changes and begin exporting for you in HD and SVG formats.

**Comment by ccfiel** [comment]
yes it's better to have no receipt also the T is a little bit farther to the other letters. The color suggestion would be pallet of blue

**Comment by dee-y** [comment]
I have measured the letter spacing and it's not so Chang than before. I also developed a new logo variation for you, now options A or B... And numbers 1 to 6...please make a pick as regards logo A or B, and which color variation you'd prefer in numbers 5a, b, or c. More contributions are greatly welcomed.

![tailpos-04](https://user-images.githubusercontent.com/36637989/58386440-de9d2100-7ff7-11e9-98c5-ea2ac972a926.jpg)
